db/migrate/20190723170759_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.1]
+  def change
+    create_table :books do |t|
+      t.string :genre
```

Comment:
It's a good idea add DB's null constraint in your table, take it as a "double safe"
```suggestion
      t.string :genre, null: false
```

Reply:
I read an [article](https://thoughtbot.com/blog/validation-database-constraint-or-both) about this and I agree, will be adding the constraints to all the required fields.

Reply:
Question. Should I create a new migration to change this or is this an exception in which I can alter the migration files?

Reply:
I decided it would be best to keep adhering to best practices and created a new migration to set the DB not null constraints. This migration can be found in the latest commit.